app/views/manuals/_manual_section.html.erb

```diff
@@ -9,7 +9,7 @@
       <% if presented_document.body.present? %>
         <div class='body-content-wrapper'>
           <%= render 'govuk_publishing_components/components/govspeak', {} do %>
-            <%= raw(presented_document.body) %>
+            <%= sanitize(presented_document.body[:intro]) %>
```

Comment:
I might be being overly cautious and this may be fine, but we need to be careful of this as it can break tables etc: check this url against your version: https://www.gov.uk/guidance/mot-inspection-manual-for-private-passenger-and-light-commercial-vehicles/3-visibility

Reply:
Aha, this does indeed break tables 😬  I'll investigate

Reply:
I think it may have been manuals where I looked into this before - we had to roll back a release where we'd broken tables on live, but when I looked into what we needed to allow on the sanitize list it was pretty much everything - probably best to leave it as raw for now.

Reply:
Makes sense, was a good spot, cheers 👍  As it happens this was mainly put in to fix a test, however upon revision I think I've come up with a solution that uses nokigiri properly and is much more readable.